src/adagrams.js

```diff
@@ -1,7 +1,156 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    let letterPool = (
```

Comment:
This should be a `const`

Reply:
Consider first creating an object that represents letter frequencies and then using this object to build your array.